app/policies/placement_policy.rb

```diff
@@ -0,0 +1,15 @@
+class PlacementPolicy < ApplicationPolicy
```

Comment:
Do you guys have any plans to move the `Placement` model into the `Placements` namespace, similar to how we have `Claims::Claim`?

Reply:
Possibly, but not as part of this PR